proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java

```diff
@@ -437,6 +437,14 @@ public void handle(PluginMessage pluginMessage) throws Exception
                     player.disconnect( new TextComponent( kickReason ) );
                 }
             }
+            if ( subChannel.equals( "Alert" ) )
+            {
+                String message = in.readUTF();
+                if ( message != null && message != "" )
```

Comment:
In Java strings should never be compared by identity, use `!message.isEmpty()` here and in general `equals()`.